h3i/src/prompts/h3/mod.rs

```diff
@@ -423,6 +429,23 @@ fn prompt_extension() -> InquireResult<Action> {
     Ok(action)
 }
 
+pub fn prompt_connection_close() -> InquireResult<Action> {
+    let is_app = prompt_yes_no("application close:")?;
```

Comment:
there's already a `prompt_transport_or_app()` to reuse

Reply:
Total brain fart, nice catch